iphone/cli/commands/_build.js

```diff
@@ -4493,6 +4493,7 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 			this.logger.info(__('Processing JavaScript files'));
 
 			async.eachSeries(Object.keys(jsFiles), function (file, next) {
+				setImmediate(function(){
```

Comment:
Last thing, then you can indent. We do spaces between `function` and `()` and `{`. The space between `function` and `(` is because we always put a space after `function` regardless if the function is named or anonymous. Since functions are no different than other control blocks such as `if` and `while`, we do a space between the `)` and `{`.